services/core-api/app/api/mines/permits/permit_amendment/models/permit_amendment.py

```diff
@@ -44,11 +47,11 @@ class PermitAmendment(SoftDeleteMixin, AuditMixin, Base):
     permit_amendment_status_description = association_proxy('permit_amendment_status',
                                                             'description')
     permit_guid = association_proxy('permit', 'permit_guid')
-    permit_no = association_proxy('permit', 'permit_no')
+    permit_no: str = association_proxy('permit', 'permit_no')                                                                                                                              #type: ignore[reportAssignmentType]
```

Comment:
This is for pylance, but i'm not sure about the details of this type hinting override.